src/main/webapp/index.html

```diff
@@ -69,5 +69,6 @@
         </ul>
       </div>
     </nav>
+    <div id="forum"></div>
```

Comment:
I think every section of HTML should be commented. So just as there is `<!-- Navbar -->` above the navbar, there should be `<!-- Forum -->` above this line.

Reply:
done